### DISCUSSION TITLE
AttributeError: module 'neurokit2' has no attribute 'ecg_invert'

### DISCUSSION THREAD

**Discussion by SHADDYHAS** [neuropsychology/NeuroKit#903] [Q&A]
Hi,
ecg_invert stopped working today and I'm getting the following error:
AttributeError: module 'neurokit2' has no attribute 'ecg_invert'

It even doesn't show up in the autocomplete list:
![image](https://github.com/neuropsychology/NeuroKit/assets/113120437/0281d703-95c0-4b0e-bebb-40983aa1ec73)

Is it related to the 0.2.6 release earlier today?

**Comment by DominiqueMakowski** [comment]
🤔 I just checked and it does work for me, are you sure you have the latest version? perhaps you reverted to an old one by error?

**Reply by SHADDYHAS** [reply]
Thank you for the quick response!
I was using the newest NeuroKit version 0.2.6 on Google Colab. 
Restarting the runtime didn't help.
Only when I installed 0.2.5 and then reinstalled the newest version, it worked. 
![image](https://github.com/neuropsychology/NeuroKit/assets/113120437/c8989bca-b2c0-43ba-953a-e3ff433dd332)

**Reply by DominiqueMakowski** [reply]
very weird, especially since we didn't change anything related to that function
